mmdet3d/core/bbox/samplers/iou_neg_piecewise_sampler.py

```diff
@@ -58,8 +58,8 @@ def _sample_neg(self, assign_result, num_expected, **kwargs):
         neg_inds = torch.nonzero(assign_result.gt_inds == 0, as_tuple=False)
         if neg_inds.numel() != 0:
             neg_inds = neg_inds.squeeze(1)
-        if len(neg_inds) <= num_expected:
-            return neg_inds
+        if len(neg_inds) <= 0:
+            return False
```

Comment:
This return is really weird here. It returns a different type. Should assert here